doc/nrf/shortcuts.txt

```diff
@@ -90,11 +90,11 @@
    You can continue to use it until the transition is complete in the |NCS| and the feature is removed in one of the upcoming |NCS| releases.
    For more information, see :ref:`zephyr:sysbuild`, :ref:`sysbuild_images`, :ref:`zephyr_samples_sysbuild`, and :ref:`sysbuild_forced_options`.
 
-.. |nrfjprog_deprecation_note| replace:: Starting with the |NCS| v2.8.0, nrfjprog is officially deprecated and will be removed in an upcoming release.
-   Transition to using `nRF Util (device command) <Device command overview_>`_ for all related tasks going forward.
+.. |nrfjprog_deprecation_note| replace:: Starting with the |NCS| v2.8.0, nrfjprog is in the process of being archived.
+   It will remain available for download, but `nRF Util (device command) <Device command overview_>`_ will gradually replace it.
```

Comment:
Is <Device command overview_> a placeholder for information?

Reply:
It's a link to the docs of nRF Util.